packages/web-runtime/src/components/Topbar/ApplicationsMenu.vue

```diff
@@ -57,6 +60,8 @@ export default {
     }
   },
   computed: {
+    ...mapGetters(['configuration', 'getToken']),
```

Comment:
Instead of `getToken` (already deprecated) please map the `runtime/auth` getter `accessToken`